Project.toml

```diff
@@ -29,10 +29,11 @@ VectorizationBase = "3d5dd08c-fd9d-11e8-17fa-ed2836048c2f"
 [weakdeps]
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+NNlib = "872c559c-99b0-510c-b3b7-b6c96a88d5cd"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 
 [extensions]
-ForwardDiffExt = ["ChainRulesCore", "ForwardDiff"]
+ForwardDiffExt = ["ChainRulesCore", "ForwardDiff", "NNlib"]
```

Comment:
NNLib should be a separate extension. Not only is this breaking, it also means that it won't properly trigger for most users.